amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -1,94 +1,177 @@
+
+. In the *Mule Palette* view, select *Amazon S3* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Amazon S3 system. Amazon S3 supports basic and role authentication.
+
+To configure the global element for Amazon S3 Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, in *Connection*, select the authentication method to configure:
+  * <<basic-authentication,Basic Authentication>>
+  * <<role-authentication,Role Authentication>>
+
 +
-. If you are:
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
 +
-. If you are using the default Amazon S3 storage, leave the default entries for *Proxy*, *Security*, and *Advanced* tabs.
-. If you are connecting to different storage than the default AWS S3, specify its URL in the *Advanced* tab's *S3 Compatible Storage URL* field.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
 
-. Click *Test Connection* to confirm that the parameters of your global configuration are accurate and that Mule is able to successfully connect to your instance of Amazon S3.
 +
 Validating a connection with *Test Connection* requires that you have permission in AWS IAM to the action `s3:ListAllMyBuckets`. If you don’t have this permission, the test fails. However, you can still use the connector and the operations to which you have access.
 +
 Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
 
-. Click *OK* to save the global connector configurations.
+[[basic-authentication]]
+=== Basic Authentication
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
```

Comment:
Strictly speaking, this should now be step 1 of an ordered list because we have a step 2 (the Advanced tab). Same for the other type of authentication.